extensions/amp-analytics/analytics-vars.md

```diff
@@ -361,3 +371,75 @@ Provides the number of seconds that have elapsed since 1970. (Epoch time)
 
 Example value: `1452710304312`
 
+## Visibility Variables
+
+### backgrounded
```

Comment:
this is not related to prerendering. Nothing in amp-analytics executes before the page is made visible afaik. Will add some context here.